documentation/source/event-model.rst

```diff
@@ -63,20 +63,22 @@ The ``Trade`` data type defines the outcome of a financial transaction between p
 .. code-block:: Haskell
 
  type Trade:
-   [metadata key]
-   tradeIdentifier TradeIdentifier (1..*)
-   tradeDate date (1..1)
-     [metadata id]
-   tradableProduct TradableProduct (1..1)
-   party Party (0..*)
-   partyRole PartyRole (0..*)
-   executionDetails ExecutionDetails (0..1)
-   contractDetails ContractDetails (0..1)
-   clearedDate date (0..1)
-     [deprecated]
-   collateral Collateral (0..1)
-   account Account (0..*)
-     [deprecated]
+	[metadata key]
+	tradeIdentifier TradeIdentifier (1..*)
+	tradeDate date (1..1)
+        [metadata id]
+	tradeDateTime zonedDateTime (0..1)
+        [metadata id]
+	tradableProduct TradableProduct (1..1)
+	party Party (0..*)
+	partyRole PartyRole (0..*)
+	executionDetails ExecutionDetails (0..1)
+	contractDetails ContractDetails (0..1)
```

Comment:
Is contracttDetails being added in this PR as well?

Reply:
No that is probably just the result of indenting.